queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM ANIMALS WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM ANIMALS WHERE weight_kg BETWEEN 10.3 AND 17.4;
```

Comment:
- [ ] Kindly check your weight figures, the weight should be between 10.4kg and 17.3kg.

Reply:
Hi @danmainah Please it says it should include that too